### PR TITLE
 Adding duration to tag context instead of sentry message

### PR DESCRIPTION
**Pull request by vrathee**
@anujsi Please review

### CODE REVIEW

**Comment by anujsi** [comment]
Reviewed.

File: src/jwt.ts [hunk 1]
```diff
@@ -1176,15 +1176,15 @@ function updateTokenSuccess(refreshed: boolean) {
...
+    let userLoginDuration = undefined;
     if (initialUserToken) {
-        userLoginTime = ((+new Date() - initialUserToken.auth_time * 1000) / 1000 / 60 / 60).toFixed(1);
+        userLoginDuration = ((+new Date() - initialUserToken.auth_time * 1000) / 1000 / 60 / 60).toFixed(1);
     }
     failCountEqualsThreshold(FAIL_COUNT_NAME, FAIL_COUNT_THRESHOLD).then((isfailCountEqualsThreshold) => {
-        if (isfailCountEqualsThreshold && userLoginTime < 14) {
-            sendToSentry(new Error(`[jwt.js] Update token failure: after ${FAIL_COUNT_THRESHOLD} attempts within ${userLoginTime} hours of logging in`), e);
+        if (isfailCountEqualsThreshold && userLoginDuration < 14) {
+            sendToSentry(new Error(`[jwt.js] Update token failure: after ${FAIL_COUNT_THRESHOLD} attempts.`), {error}, {userLoginDuration});
```
Comment (rh-kgarg) [inline src/jwt.ts]: can we keep the tag key as user_login_duration instead of userLoginDuration to be consistent as L1520

**Comment by rh-kgarg** [comment]
Just 1 comment. Rest looks good

File: src/jwt.ts (hunk 1 above)
Comment (vrathee) [reply src/jwt.ts]: yes.. would be good!

**Pull request merged**